### DISCUSSION TITLE
web browser link with dir tree auto load

### DISCUSSION THREAD

**Discussion by cbhanni** [avelino/awesome-go#4993] [Ideas]
I would be cool if i could load all of this into chrome, Firefox, ... fave web browser. awesome_go user imports for chrome. It would help if i had to jump on to some system and want to load this dir tree in my book marks. might already be here, thanks :).

**Comment by avelino** [comment]
@cbhanni I don't understand your need, can you elaborate on the idea?

do you know [awesome-go.com](https://awesome-go.com)?
